examples/kubernetes-local-redirect/lrp-svcmatcher.yaml

```
@@ -0,0 +1,21 @@
+apiVersion: "cilium.io/v2"
+kind: CiliumLocalRedirectPolicy
+metadata:
+  name: "lrp-svc"
+spec:
+  redirectFrontend:
+    serviceMatcher:
```

nit: We could make this a little more consistent with other similar constructs in CNP:

```suggestion
    toServices:
```

---

I had +1, but then edited the comment. Idk, if that's the case, why don't we also have a `toAddress` instead?